net-mgmt/zabbix-agent/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		zabbix-agent
-PLUGIN_VERSION=		1.10
+PLUGIN_VERSION=		1.11
```

Comment:
1.11 might be a little drastic for a help text change. Usually I avoid bumping version/revision altogether in these cases but it's up to you

Reply:
Agreed. :) I hope to add more before merging it. But if that doesn't happen, I'll revert the version bump.